model/nh_core.F90

```diff
@@ -206,15 +206,15 @@ subroutine Riem_Solver3(ms, dt,   is,   ie,   js, je, km, ng,    &
                             kapad2,  &
 #endif
                             pe2, dm,   &
-                            pm2, pem, w2, dz2, pt(is:ie,j,1:km), ws(is,j), p_fac)
+                            pm2, pem, w2, dz2, pt(is:ie,j,1:km), ws(is,j), p_fac, tau_w)
       else
            call SIM_solver(dt, is, ie, km, rdgas, gama, gm2, cp2, akap, &
 #ifdef MULTI_GASES
                            kapad2,  &
 #endif
                            pe2, dm,  &
                            pm2, pem, w2, dz2, pt(is:ie,j,1:km), ws(is,j), &
-                           a_imp, p_fac, scale_m)
+                           a_imp, p_fac, scale_m, tau_w)
```

Comment:
tau_w should be fast_tau_w_sec added in nh_core.F90, nh_utils.F90 ?

Reply:
Hi, Kate. I believe that inside the nh_core/nh_utils modules the name is `tau_w` and the namelist flag `fast_tau_w_sec` is used in the call to the NH utils. So this implementation is correct, but potentially confusing. However for consistency's sake and to eliminate confusion we could replace this with `fast_tau_w_sec`.

Thanks,
Lucas

Reply:
I will update this also to use fast_tau_w_sec as well. You are right that this could be confusing to follow.